software/wmla120.py

```diff
@@ -2059,6 +2060,29 @@ def run_ansible_task(self, yamlfile):
         for task in install_tasks:
             if 'engr_mode' in task['tasks'] and not self.eng_mode:
                 continue
+            if task['description'] == "PowerAI tuning recommendations":
+                no_reboot_hosts = (
+                    get_host_list_no_reboot(self.sw_vars['ansible_inventory']))
+                if len(no_reboot_hosts) > 0:
+                    print(bold("\nInstallation cannot complete until all "
+                               "clients have been rebooted."))
+                    print(bold("\nThe following client nodes have not been "
+                               "automatically rebooted: "))
+                    for host in no_reboot_hosts:
+                        print(f"    {host}")
+                    print(bold("\nPlease manually reboot these hosts and then "
+                               "run the following command from the "
+                               "installer:"))
+                    tasks_path = f'{self.my_name}_ansible/' + task['tasks']
+                    cmd = (f'{get_ansible_playbook_path()} -i '
+                           f'{self.sw_vars["ansible_inventory"]} '
+                           f'{GEN_SOFTWARE_PATH}'
+                           f'{self.my_name}_ansible/run.yml '
+                           f'--extra-vars '
+                           f'"task_file={GEN_SOFTWARE_PATH}{tasks_path}" '
+                           f'--ask-become-pass\n')
+                    print(f"    {cmd}")
```

Comment:
Why not just write this command to a bash script in the power-up/scripts directory and then instruct them to run the command? We could also include the commands to restart the spectrum services;
sudo bash -c 'source /opt/ibm/spectrumcomputing/profile.platform && egosh ego start'